test/integration/targets/azure_rm_managed_disk/tasks/main.yml

```diff
@@ -59,6 +61,45 @@
      that:
        - output.changed
        - output.state.id is defined
+       - output.state.os_type == None
+
+ - name: Change the operating system type of the managed disk to Linux
+   register: output
```

Comment:
Please move the register to the end of the task

Reply:
Done in the latest push - sorry I forgot about that earlier.